doc/userguide/rules/smtp-keywords.rst

```diff
@@ -34,7 +34,7 @@ smtp.command_line
 A single line from the client to the server. Multi-line commands will have a frame per
 line. Lines part of the SMTP DATA transfer are excluded.
 
-.. container:: example fule
+.. container:: example-rule
```

Comment:
Couldn't find yet a way for sphinx to figure out that there's something wrong in a case like this. Would it make sense to have a script to check whether a CSS container really exists in our suricata.css file? Could be run in one of the workflows that checks our docs.